pyproject.toml

```diff
@@ -1,19 +1,21 @@
 [tool.poetry]
 name = "snakemake-storage-plugin-gcs"
-version = "0.1.3"
+version = "0.1.3.2"
 description = "A Snakemake storage plugin for Google Cloud Storage"
 authors = [
     "Vanessa Sochat <sochat1@llnl.gov>",
     "Johannes Koester <johannes.koester@uni-due.de>"
 ]
 readme = "README.md"
-repository = "https://github.com/snakemake/snakemake-storage-plugin-gcs"
+repository = "https://github.com/jairav/snakemake-storage-plugin-gcs"
```

Comment:
Could you please fix this back to be snakemake?

Reply:
Done!